cmd/cloud/installation.go

```diff
@@ -615,3 +620,92 @@ var installationRecoveryCmd = &cobra.Command{
 		return nil
 	},
 }
+
+var installationDeploymentReportCmd = &cobra.Command{
```

Comment:
I'm a little curious why you chose to make this a new command instead of a flag like `--table`